ui-guidelines/index.html

```diff
@@ -162,27 +162,27 @@ <h4 class="u-move-down-xlarge">Basic buttons</h4>
 								</div>
 								<div class="button-wrapper">
 									<div class="button-box">
-										<p class="button-card-header">Secondary button</p>
+										<p class="button-card-header">Primary button</p>
 										<div class="button-card-content">
-											<button type="button" class="ez-button ez-button-secondary">Save</button>
+											<button type="button" class="ez-button ez-button-primary">Confirm</button>
 										</div>
 									</div>
 									<div class="button-code">
 <pre>
-<code class="language-markup"><script type="prism-html-markup"><button type="button" class="ez-button ez-button-secondary">Save</button></script></code>
+<code class="language-markup"><script type="prism-html-markup"><button type="button" class="ez-button ez-button-primary">Confirm</button></script></code>
 </pre>
 									</div>
 								</div>
 								<div class="button-wrapper">
 									<div class="button-box">
-										<p class="button-card-header">Neutral button</p>
+										<p class="button-card-header">Secondary button</p>
 										<div class="button-card-content">
-											<button type="button" class="ez-button ez-button-neutral">Cancel</button>
+											<button type="button" class="ez-button ez-button-secondary">Save</button>
 										</div>
 									</div>
 									<div class="button-code">
 <pre>
-<code class="language-markup"><script type="prism-html-markup"><button type="button" class="ez-button ez-button-neutral">Cancel</button></script></code>
+<code class="language-markup"><script type="prism-html-markup"><button type="button" class="ez-button ez-button-secondary">Save</button></script></code>
```

Comment:
it's already done in the guideline
![ui](https://cloud.githubusercontent.com/assets/305563/26394531/8b871802-406d-11e7-8c7a-6206724e2565.png)